README.md

```diff
@@ -134,9 +134,9 @@ controller_server:
   ros__parameters:
     FollowPath:
       plugin: "mppi::Controller"
-      time_steps: 15
-      model_dt: 0.1
-      batch_size: 300
+      time_steps: 40
+      model_dt: 0.075
```

Comment:
why not 0.1 ? Is there much difference with lower wz_std ?

Reply:
I noticed smoothness improvements with this lowered, but wanted to find a balance between what was computationally reasonable and what would give us the "most bang for our buck" in terms of smoothness. The best value I found was actually 0.05, but it was impractical to do that with 60 timesteps and 2000+ batches. So I increased it to 0.075 as a trade-off.

But with the 50hz/1000 batch for @padhupradheep's recommendations, I can re-evaluate this if its important and back it off to 0.1/30 steps if it doesn't make much of a difference with those settings